packages/shared/src/modules/activities/libs/types/activity-get-activities-response-dto.type.ts

```diff
@@ -0,0 +1,9 @@
+import { type ValueOf } from "../../../../libs/types/types.js";
+import { type ActivityTypeValue } from "../enums/enums.js";
+import { type ActivityResponseDto } from "./activity-response-dto.type.js";
+
+type ActivityGetActivitiesResponseDto = {
```

Comment:
maybe sth like this will be more readable
```suggestion
type ActivityGetAllResponseDto = {
```